Std/Data/Char.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2022 Jannis Limperg. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jannis Limperg
+-/
+
+namespace Char
+
+theorem csize_eq (c) :
+    String.csize c = 1 ∨ String.csize c = 2 ∨ String.csize c = 3 ∨
+    String.csize c = 4 := by
+  simp only [String.csize, utf8Size]
+  repeat (first | split | (solve | simp))
```

Comment:
Let's make this one private for now. I would like to rewrite it later to remove it in favor of the two following theorems.

Reply:
Done.